### PR TITLE
[Tabs] Refresh the show more dropdown

### PR DESCRIPTION
**Pull request by buttersnipps**
Fixes Issue: #6013 
Fixed to match the desired css.

### Summary of Changes
Changed some properties in Dropdown.css

### Screenshots/Videos (OPTIONAL)
For small names -
screen shot 2018-04-20 at 2 15 23 pm

For longer names , add scroll bar-

screen shot 2018-04-20 at 2 14 44 pm

### CODE REVIEW

File: src/components/shared/Dropdown.css [hunk 1]
```diff
@@ -6,6 +6,7 @@
   --width: 150px;
   background: var(--theme-body-background);
   border: 1px solid var(--theme-splitter-color);
+  border-radius: 10px;
```
Comment (jasonLaster) [inline src/components/shared/Dropdown.css]: i think this should be closer to 4px

File: src/components/shared/Dropdown.css [hunk 2]
```diff
@@ -44,6 +45,7 @@ html[dir="rtl"] .dropdown {
   overflow: hidden;
   height: 30px;
   text-overflow: ellipsis;
+  width: fit-content;
```
Comment (jasonLaster) [inline src/components/shared/Dropdown.css]: nice

File: src/components/shared/Dropdown.css (hunk 1 above)
Comment (buttersnipps) [reply src/components/shared/Dropdown.css]: Ok. I'll make the adjustments.

File: package.json [hunk 3]
```diff
@@ -75,7 +75,6 @@
     "devtools-splitter": "^0.0.6",
     "devtools-utils": "^0.0.12",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint-plugin-file-header": "0.0.1",
     "eslint-plugin-import": "^2.8.0",
```
Comment (bomsy) [inline package.json]: not part of this work but maybe we move this to devDependencies as well .. thanks

File: package.json [hunk 4]
```diff
@@ -119,6 +118,7 @@
     "eslint": "^4.2.0",
     "eslint-config-prettier": "^2.3.0",
     "eslint-plugin-babel": "^5.0.0",
+    "eslint-plugin-file-header": "0.0.1",
```
Comment (bomsy) [inline package.json]: thanks!

**Comment by jasonLaster** [comment]
small thing, but can you update the PR title?

**Comment by buttersnipps** [comment]
@darkwing Hey, for bigger file names it wasn't fitting the entire name. So I added it to fit those bigger file names But if it gives problems , it can be removed.

**Comment by darkwing** [comment]
@buttersnipps Interesting.  What about using:

```css
.whatever-selector li {
  overflow-x: hidden;
  text-overflow: ellipsis;
}
```

That may be a better option to avoid weirdness.

**Comment by buttersnipps** [comment]
@darkwing Your suggestion did not work as desired but using `display: inline-block` did work.

**Pull request merged**